petsc/install_petsc.sh

```diff
@@ -10,16 +11,22 @@ PYTHON=/usr/bin/python3.11
 BASE_INSTALL_DIR=$PWD
 MAKE_NP=32 # Use up to 32 cores when building
 
-# Clone Firedrake fork of repositories
-git clone https://github.com/firedrakeproject/petsc.git
-git clone https://github.com/firedrakeproject/slepc.git
+# Clone PETSc and SLEPc repositories
+git clone https://gitlab.com/petsc/petsc.git
+git clone https://gitlab.com/slepc/slepc.git
```

Comment:
We now just pass `--download-slepc` to PETSc configure.

Reply:
It's built separately so that Firedrake can be installed with or without SLEPc support. I followed up on this (and points 1 & 2) in a video call.